CHANGES.rst

```diff
@@ -1,5 +1,6 @@
-32-cvs (in development)
-=======================
+=========================
+ 32-cvs (in development)
+=========================
```

Comment:
I'm not sure it was necessary to restyle every headings.  But I guess it's your mode that made the change sneakily?

Reply:
Hmm, yes, I didn't even catch that myself.

Reply:
Fixed